docker/etc/profile.d/treeherder.sh

```diff
@@ -0,0 +1,22 @@
+# Mapping is taken from docker link assumptions
```

Comment:
Would be good to consolidate some of this, https://github.com/mozilla/treeherder-service/blob/master/puppet/manifests/vagrant.pp#L39 and https://github.com/mozilla/treeherder-service/blob/master/puppet/manifests/production.pp#L40 at some point in the future.